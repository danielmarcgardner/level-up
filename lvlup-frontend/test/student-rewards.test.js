import StudentRewards from '../src/components/Student/student-rewards/student-rewards-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

test('StudentRewards should render', () => {
  const reward = shallow(
    <StudentRewards />,
  );
  expect(toJson(reward)).toMatchSnapshot();
});
