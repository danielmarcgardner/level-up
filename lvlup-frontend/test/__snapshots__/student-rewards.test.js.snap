exports[`test StudentRewards should render 1`] = `
<div
  className="lvl-table">
  <Connect(RewardsTable) />
</div>
`;
